spec/spec.md

```diff
@@ -28,24 +28,27 @@ Participate:
 ------------------------------------
 
 ## Abstract
+  
+The JWT VC Presentation Profile defines a standardized way to interoperably exchange [[ref:Verifiable Credentials]] (VCs) between digital [[ref: Wallets]] and credential [[ref: Verifiers]].
 
-The JWT VC Presentation Profile defines a set of specifications to enable the interoperable presentation of Verifiable Credentials between Wallets and Verifiers/RPs.
+This document is **specification profile** not a specification. It is a collection of pre-existing specficiations with all but required optionality stripped from them with the purpose of ensuring interoperability.
+
+Currently, there are many published specifications for exchanging VCs between [[ref:Wallets]] and [[ref:Verifiers]]. The Wallet and Verifier providers have to choose which standards to operate under and hope that the external parties they wish to exchange VCs with support those same standards. Specifications tend to be very broad, flexible, and un-opinionated. This is an advantage when building use cases and providing optionality to the implementers. But, in the case of exchanging VCs, it can cause misalignment. The more optionality in the specification, the more complex and expensive it becomes to deliver code, maintain a solution, and verify a VC. All of this complexity decreases the liklihood of successful interop.
+
+The JWT VC Presentation Profile, once adopted by providers, enables the seamless verification of VCs. It is based on [[ref: OpenID4VP]] which leverages the existing infrastructure and security of [[ref: Open ID Connect (OIDC)]] and OAuth 2.0 and reduces implementation cost for existing OIDC Relying Parties to become VC verifiers. The profile relies on [[ref: Presentation Exchange]] for the presentation of VCs, [[ref:Well Known DIDs]] for trust establishment, and [[ref:Status List 2021]] for revocation of VCs.
```

Comment:
I think most of what is in the top paragraph is true, but I am not comfortable including it this bluntly...

PS Why do we keep referencing Wallets and Verifiers, etc? I think it is distracting and makes it harder to read a text.

```suggestion
This profile is needed because currently, there are multiple specifications available across each layer of a technical stack used to exchange VCs between Wallet and Verifiers. This leads to implementers needing to choose which specifications to combine, without a guaranteed interoperability. 

This profile is based on [[ref: OpenID4VP]] which leverages the existing infrastructure and security of [[ref: Open ID Connect (OIDC)]] and OAuth 2.0 and reduces implementation cost for existing OIDC Relying Parties to become VC verifiers. The profile relies on [[ref: Presentation Exchange]] for the presentation of VCs, [[ref:Well Known DIDs]] for trust establishment, and [[ref:Status List 2021]] for revocation of VCs.
```

Reply:
I am tempted to say we do not need this last paragraph, and we can instead just say "for the open standards composing this profile, see Overview of the Open Standards Requirements Section".

Reply:
Agree w/ @Sakurann's suggested text here, and I'd lean towards having the second paragraph as a gentle introduction for those who TLDR it right after the abstract.

Reply:
It might be too "wordy" but I thought it was a nice overview for people who didn't want to read the rest.... Maybe it needs a subsection instead below? or i can remove all together...

Reply:
also, curious why you are not comfortable with the top paragraph? something about how things were written before was not clear to readers (some of the #34 discussion). so i think we need a little bluntness